### PR TITLE
Change returning type of executeTx

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

+  `executeTx` method from `WebMode` class now returns `Promise<algosdk.modelsv2.PendingTransactionResponse>`.

### CODE REVIEW

File: packages/web/src/lib/web-mode.ts [hunk 1]
```diff
@@ -38,7 +40,7 @@ export class WebMode {
 				pendingInfo[CONFIRMED_ROUND] !== null &&
 				(pendingInfo[CONFIRMED_ROUND] as number) > 0
 			) {
-				return pendingInfo;
+				return pendingInfo as unknown as algosdk.modelsv2.PendingTransactionResponse;
```
Comment (vuvth) [inline packages/web/src/lib/web-mode.ts:43]: Can we only use as `algosdk.modelsv2.PendingTransactionResponse` at here?

Comment (sczembor) [reply packages/web/src/lib/web-mode.ts:43]: No we can't, there are some properties missing

File: packages/web/src/lib/web-mode.ts [hunk 2]
```diff
@@ -129,18 +131,22 @@ export class WebMode {
...
+	): Promise<algosdk.modelsv2.PendingTransactionResponse> {
 		let txns: Transaction[] = [];
-		if (execParams.length > 16) {
+		if (transactions.length > 16) {
 			throw new Error("Maximum size of an atomic transfer group is 16");
 		}
-
-		for (const [_, txn] of execParams.entries()) {
-			txns.push(mkTransaction(txn, await this.getSuggestedParams(txn.payFlags)));
+		if (!isSDKTransactionAndSign(transactions[0])) {
```
Comment (robert-zaremba) [inline packages/web/src/lib/web-mode.ts:144]: add a check before the if:
```
if (transactions.length == 0)
   throw new SomeErrorClass("at least one transaction must be provided")
```

Please find the best error for the error class from the errors we have defined.

Comment (sczembor) [reply packages/web/src/lib/web-mode.ts:147]: done

File: packages/web/src/lib/web-mode.ts [hunk 3]
```diff
@@ -38,7 +41,7 @@ export class WebMode {
 				pendingInfo[CONFIRMED_ROUND] !== null &&
```
Comment (vuvth) [inline packages/web/src/lib/web-mode.ts:41]: How about only waiting 10 or 6 round?
We defined [WAIT_ROUNDS](https://github.com/scale-it/algo-builder/blob/develop/packages/web/src/lib/constants.ts#L4) in constants.ts

**Review by vuvth — APPROVED** [review]
lgtm! Just have on suggestion!

**Pull request merged**